### ISSUE TITLE
Bug: rewrite start or end whitespace control in statement block

### ISSUE THREAD

**Issue by CrazyUmka** [davidodenwald/prettier-plugin-jinja-template#27]
When you try to add whitespace control like this:
```jinja
{%- if  %}
hello world
{% endif %}
```
After reformatted you get the same whitespace control in the end.
```jinja
{%- if  -%}
hello world
{% endif %}
```

Documentation reference: https://jinja.palletsprojects.com/en/3.1.x/templates/#whitespace-control

**Comment by davidodenwald** [comment]
This is actually done deliberately to enforce a code style where the start and end delimiters should be the same.

**Comment by cypressSap** [comment]
Love the plugin aside from this bug! But it *is* a bug, not a style preference: [Prettier's first requirement](https://prettier.io/docs/en/rationale#correctness) is to output code "that has the exact same behavior as before formatting," and changing `{%- if %}` to `{%- if -%}` changes whether whitespace is stripped after the tag.

**Comment by davidodenwald** [comment]
I always thought that `{%- if %}` and `{%- if -%}` did exactly the same thing, but after reading the docs again I saw the different functions it triggers.

I will fix the issue with the next release.

**Comment by davidodenwald** [comment]
Sorry for the long wait. I published [v1.4.1](https://github.com/davidodenwald/prettier-plugin-jinja-template/releases/tag/v1.4.1) with the fix.

**Comment by cypressSap** [comment]
Works great now, thank you!
